docs/plone-deployment/deploy.md

```diff
@@ -32,13 +32,15 @@ Also use this command when there is a new version of any of the images.
 make status
 ```
 
+Once everything is running, and if your deployment was local (with Vagrant), you will be able to visit your website at `https://plone-conference.localhost`.
+
 ### Check Logs
 
-|Tool|Command|
-|-|-|
-|webserver|`make logs-webserver`|
-|frontend|`make logs-frontend`|
-|backend|`make logs-backend`|
+| Tool      | Command               |
+| --------- | --------------------- |
+| webserver | `make logs-webserver` |
+| frontend  | `make logs-frontend`  |
+| backend   | `make logs-backend`   |
```

Comment:
Please revert the table formatting.

Reply:
@danalvrz I think the point here is to optimize the markup for easy editing, rather than for viewing the docs in text mode.

Reply:
@danalvrz If you didn't change this intentionally, you might have an extension in your editor that is autoformatting Markdown when you save.

Reply:
@davisagli yes! I recently changed some formatting settings in my editor, and I did not notice the autoformatting there. I'll revert it.